crafters/image/ImageTorchTransformation/__init__.py

```diff
@@ -0,0 +1,165 @@
+from typing import Dict, List, Union
+
+import numpy as np
+import torch
+
+from jina.executors.crafters import BaseCrafter
+
+
+class ImageTorchTransformation(BaseCrafter):
+    """Apply torchvision transforms on image batches.
+
+    This crafter creates a `Compose` transform using the list of transforms provided.
+    The numpy input is converted to tensor before applying the transform. Hence `ToTensor()` is not required.
+
+    Note: Not recommended to use inherently random transforms and
+    make sure not to use transforms that do not support tensor inputs.
+
+    Example:
+        .. code-block:: python
+            from jina.hub.crafters.image.ImageTorchTransformation import ImageTorchTransformation
+            import numpy
+
+            transforms = [
+                {"CenterCrop": dict(size=(300))},
+                {"Resize": dict(size=(244, 244))},
+                "RandomVerticalFlip",
+                {"Normalize": dict(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225))},
+            ]
+
+            crafter = ImageTorchTransformation(transforms)
+            crafted_imgs = crafter.craft(image_batch)
+
+        The torch equivalent for this would be recursively apply `Compose` on each image.
+
+        .. code-block:: python
+            import torchvision.transforms as T
+
+            transforms = T.Compose(
+                [
+                    T.ToTensor(),
+                    T.CenterCrop(300),
+                    T.Resize((244, 244)),
+                    T.RandomVerticalFlip(p=1.0),
+                    T.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
+                ]
+            )
+
+            crafted_img = transforms(image_batch[0]).cpu().numpy()
+            crafted_img = np.transpose(crafted_img, (1, 2, 0))
+
+
+    :param transforms: List of transforms that are applied sequentially
+    :param args:  Additional positional arguments
+    :param kwargs: Additional keyword arguments
+    """
+
+    def __init__(
+        self,
+        transforms: List[Union[str, Dict[str, Dict]]],
```

Comment:
can we have a default value where a Normalization and Resize is applied?
The typical Resize and normalization required for the main Convolutional Neural Network models?

Reply:
> Can you please use single quote? as per style?

I saw most code was single quotes but I also saw the that the style is Black, which enforces double quotes everywhere right? 

So should I change everything to single except for long strings?

Reply:
change it everywhere, even for long strings

Reply:
> can we have a default value where a Normalization and Resize is applied?
> The typical Resize and normalization required for the main Convolutional Neural Network models?

Sure, I shall try to match it to ResNet dimensions and Normalize with ImageNet stats, as its mostly used pre-trained nets